src/api/rif-storage-upload-service/upload/index.ts

```diff
@@ -67,8 +70,8 @@ export default class UploadService
           id: 'service-post',
         })
       }
-      const data = await response.json()
+      const data: UploadResponse = await response.json()
 
-      return data.fileHash
+      return data
```

Comment:
don't need to await here as the return type is already a Promise
```suggestion
      return response.json()
```

Reply:
No actually just forgot to map it